documentation/source/users/cantherm/input.rst

```diff
@@ -131,18 +139,17 @@ Parameter               Required?                   Description
 ``rotors``              optional                    A list of :class:`HinderedRotor()` and/or :class:`FreeRotor()` objects describing the hindered/free rotors
 ======================= =========================== ====================================
 
-The ``atom`` and ``bond`` parameters are used to apply atomization energy corrections (AEC), bond corrections (BC), and spin orbit corrections (SOC) for a given ``modelChemistry()`` (see `Model Chemistry`_).
-
-Allowed atom symbols for the ``atoms`` parameter are
-``'C'``, ``'N'``, ``'O'``, ``'S'``, ``'P'``, and ``'H'``. For example, for formaldehyde we would write::
```

Comment:
We may still want to leave some description in the documentation of the atoms we commonly have AEC for

Reply:
Ah, I wasn't sure what you meant by this earlier, because the information about which AECs are available is still in the table, which I didn't delete. You mean we should have a description about which atomic energies are available for the implemented model chemistries, correct? I'll add that.

Reply:
Yes, something like what elements Canthem supports (from the atomHf dictionary you elaborated). But in second though, each modelChemistry supports slightly different elements... Perhaps we could list the common ones?
And more importantly, what do you think of outputting an Error is we don't have one of the elements within the `atomEnergies` dictionary for the selected modelChemistry (or within the user inputted atomEnergies values)?

Reply:
I don't think we should output that error because if the user is only interested in kinetics and doesn't care about the thermo, then it doesn't matter if atomic energies are available or not because they will cancel out between the products and reactants. Cantherm already prints a warning if that is the case, so at least the user would be aware of the issue.